shelley/chain-and-ledger/formal-spec/transactions.tex

```diff
@@ -86,7 +86,7 @@ \section{Transactions}
       \\
       \var{md}
       & \MetaData
-      & \N \mapsto \MetaDatum
```

Comment:
i think it is worth it to be explicit here with indexing

Reply:
One drawback is that the `\mapsto` type includes partial maps, so we could for example have the map `{0: foo, 99: bar}`. If that's not intended here, then I'd suggest at least a comment saying as much.

Reply:
even though `N -> MetaDatum` is a restriction of `MetaData`, we decided to impose this restriction. The keys are better thought of as labels, and the idea is to make sure that everyone has a mechanism for changing their schema.